int32[3] input_numbers
---
int32 greatest_common_divisor